node_modules/oae-version/lib/rest.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var OAE = require('oae-util/lib/oae');
+
+var VersionAPI = require('./api');
+
+/**
+ * @REST getVersion
+ *
+ * Get the version of OAE
+ *
+ * @Server      admin,tenant
+ * @Method      GET
+ * @Path        /version
+ * @Return      {string}                          The currently running version
+ * @HttpResponse                    200           The version will be returned
+ */
+OAE.tenantRouter.on('get', '/api/version', getVersion);
+OAE.globalAdminRouter.on('get', '/api/version', getVersion);
+
+function getVersion(req, res) {
+    VersionAPI.getVersion(function(version) {
+        return res.status(200).send(version);
```

Comment:
I think we outputting proper JSON here, but the getHilaryVersion + the documentation here confused me. Can you update the docs?